browser/ui/views/constants.cc

```diff
@@ -0,0 +1,13 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/ui/views/constants.h"
+
+namespace brave {
+
+const SkColor kBadgeNotificationBG = SkColorSetRGB(0xfb, 0x54, 0x2b);
+const SkColor kBadgeTextColor = SK_ColorWHITE;
```

Comment:
Maybe just have these as constants in the brave_icon_with_badge_image_source header file? My concern is adding another place to define color constants when we already have things at brave/ui/color/